scripts/package.json

```diff
@@ -3,16 +3,16 @@
   "version": "7.0.0-alpha.16",
   "private": true,
   "scripts": {
+    "docs:prettier:check": "cd ../docs && prettier --check ./snippets",
```

Comment:
Not sure why this change happened in the PR but I guess it makes sense, with the scripts being ordered alphabetically